### PR TITLE
column addition

### PR DESCRIPTION
**Pull request by Ansu4445**

### CODE REVIEW

File: getSQL/settings.py [hunk 1]
```diff
@@ -88,7 +88,7 @@
         'HOST': '127.0.0.1',
         'PORT': '3306',
         'USER': 'root',
-        'PASSWORD': '',
+        'PASSWORD': 'Healthy@123',
```
Comment (ashishmd) [inline getSQL/settings.py]: Can you reset the password? Else one of us have to keep on updating this file everytime we need to run the app. 

I cant do it coz i have other dependencies on mysql.

File: app/utils/import_export/read_csv.py [hunk 2]
```diff
@@ -9,3 +9,13 @@ def read_table_names(file_path):
             table_names.append(line)
         csvFile.close()
     return table_names
+
+def read_column_names(file_path_column):
```
Comment (ashishmd) [inline app/utils/import_export/read_csv.py]: We dont need this method. Above method can be reused after refactoring / renaming with a generic read method.

File: getSQL/settings.py (hunk 1 above)
Comment (Ansu4445) [reply getSQL/settings.py]: ok changed

File: app/utils/import_export/read_csv.py (hunk 2 above)
Comment (Ansu4445) [reply app/utils/import_export/read_csv.py]: yeah

File: app/utils/import_export/importer.py [hunk 3]
```diff
@@ -1,14 +1,37 @@
...
         table_data.save()
     return "Import was success."
+
+def import_columns():
+    column_names = read_csv.read_table_names(FILE_PATH_COLUMN)
+    without_foreign = [x for x in column_names if not x["foreign_key_table_name"]]
+    with_foreign = [x for x in column_names if x["foreign_key_table_name"]]
+    for column in without_foreign:
+        ##table_id = for  Tables.table_name='tablename'Tables.object.raw(id )
+        table_id = Tables.object.all(table_name='table_name').Tables.object.get(id)
```
Comment (ashishmd) [inline app/utils/import_export/importer.py]: Tables.objects.get(table_name=column["table_name"]).id

File: test_files/test_connection.py [hunk 4]
```diff
@@ -21,6 +21,7 @@ def create_connection():
 
 
 def main():
+    test = 1234
```
Comment (ashish-md) [inline test_files/test_connection.py]: remove it.

File: test_files/test_connection.py [hunk 5]
```diff
@@ -1,7 +1,7 @@
 import mysql.connector
 from mysql.connector import errorcode
 from test_files.test_csv_read import read_file
-
+import pdb; pdb.set_trace()
```
Comment (ashish-md) [inline test_files/test_connection.py]: remove this also.

File: app/views.py [hunk 6]
```diff
@@ -16,7 +16,7 @@ def welcome(request):
 # It will check for db, and if found, will return success message.
 def connection(request):
     try:
-        cnx = mysql.connector.connect(user='root', database='get_sql')
+        cnx = mysql.connector.connect(user='root', database='get_sql', password='Healthy@123')
```
Comment (ashish-md) [inline app/views.py]: remove this also if not required.

File: app/utils/import_export/importer.py [hunk 7]
```diff
@@ -1,14 +1,34 @@
...
-
-FILE_PATH = "test_files/test.csv"
-
+TABLE_FILE_PATH = "test_files/test.csv"
+COLUMN_FILE_PATH = "test_files/column_test_import.csv"
 
 # @todo : currently we are importing test file. Later we will implement upload option.
 def import_tables():
-    table_names = read_csv.read_table_names(FILE_PATH)
+
```
Comment (ashish-md) [inline app/utils/import_export/importer.py]: is table import working?

File: app/models.py [hunk 8]
```diff
@@ -0,0 +1 @@
+
```
Comment (ashish-md) [inline app/models.py]: unnecessary..

File: app/migrations/0001_initial.py [hunk 9]
```diff
@@ -20,3 +20,17 @@ class Migration(migrations.Migration):
             ],
         ),
     ]
+    operations = [
```
Comment (ashish-md) [inline app/migrations/0001_initial.py]: why is this added again?

Comment (Ansu4445) [reply app/migrations/0001_initial.py]: removed

File: app/models.py (hunk 8 above)
Comment (Ansu4445) [reply app/models.py]: removed

File: app/utils/import_export/importer.py (hunk 7 above)
Comment (Ansu4445) [reply app/utils/import_export/importer.py]: fixed now

File: app/views.py (hunk 6 above)
Comment (Ansu4445) [reply app/views.py]: done

File: test_files/test_connection.py (hunk 5 above)
Comment (Ansu4445) [reply test_files/test_connection.py]: done

File: test_files/test_connection.py (hunk 4 above)
Comment (Ansu4445) [reply test_files/test_connection.py]: done

File: app/models/__init__.py [hunk 10]
```diff
@@ -1 +1,2 @@
-from .table import Tables
\ No newline at end of file
+from .table import Tables
+from.column import Columns
```
Comment (ashishmd) [inline app/models/__init__.py]: i think there is a space issue. Check again.

File: app/utils/import_export/importer.py [hunk 11]
```diff
@@ -1,14 +1,34 @@
+from typing import Any
```
Comment (ashishmd) [inline app/utils/import_export/importer.py]: what import is this? 😂

Comment (Ansu4445) [reply app/utils/import_export/importer.py]: Honestly its not me its the bug

Comment (Ansu4445) [reply app/utils/import_export/importer.py]: removed

File: app/models/__init__.py (hunk 10 above)
Comment (Ansu4445) [reply app/models/__init__.py]: done
